content/engineering-practices-fishbowl-dev-cop.md

```diff
@@ -0,0 +1,42 @@
+Title: Fishbowl Dev CoP
+Date: 2023-12-14
+Category: Posts 
+Tags: engineering, technical-excellence, dev-cop, fishbowl
+Slug: engineering-practices-fishbowl-dev-cop
+Author: Martin M. Lacey
+Summary: Experimenting with a New Developer CoP Format: Fishbowl Discussions
+
+In this week's podcast, we're diving into a fresh approach for our Developer Community of Practice (Dev CoP). Over the years, we’ve leaned on standard presentations and open dialogue formats, but this time, we’re shaking things up with a fishbowl discussion. As part of our Scalable Agile Framework for the Enterprise (SAFe) practices at WorkSafeBC, this experiment aligns with our commitment to fostering innovation and engagement in our CoP ceremonies.
+
+# What Are Communities of Practice?
+
+Communities of Practice (CoPs) are groups of professionals united by a shared interest in a technical or business domain. They collaborate to exchange knowledge, refine their skills, and push the boundaries of their field. This concept, outlined by Etienne Wenger in Communities of Practice: Learning, Meaning, and Identity, emphasizes learning through collaboration and shared experiences.
+
+For more, see: [Communities of Practice - Scaled Agile Framework](https://scaledagileframework.com/communities-of-practice/) and [Our Technical Blog](https://wsbctechnicalblog.github.io/ceremony-community-of-practice.html).
+
+# What is a Fishbowl?
+
+In a fishbowl discussion, the “bowl” is formed by a group of guest speakers seated in a circle, engaging in a moderated discussion. Surrounding them, the audience observes in concentric rings. For open fishbowls, one chair in the inner circle remains empty, inviting audience members to step in and contribute. It’s an engaging, participatory format that encourages dynamic conversations and fresh perspectives.
+
+# Adapting Fishbowl for Hybrid Teams
+
+Post-pandemic realities mean our teams are split between remote and in-office work, posing unique challenges for interactive formats like the fishbowl. To bridge this gap, we’re piloting a hybrid model. Picture this: the core participants—our “bowl”—will be on-site, surrounded by a physical audience. Simultaneously, virtual participants will join in via Microsoft Teams, forming a “virtual outer ring.” This hybrid setup allows everyone to engage, regardless of location.
+
+Our inaugural Dev CoP Fishbowl is scheduled for a day when many engineering teams, including mine, are in the office. Limited in-person seating will make attendance a coveted experience, promoting active participation. For those online, a large monitor will provide a closer view of the live interactions, ensuring inclusivity and collaboration.
+
+>
+> ![Fishbowl Format](../images/engineering-practices-fishbowl-dev-cop-1.png)
```

Comment:
Why does everyone in this image look as if they are under 30? ;-)

Reply:
'cause everyone else has retired to a remote island to chill :)